tools/update.ps1

```diff
@@ -190,7 +190,7 @@ echo "Copying windows files for SmartSync from $WINDOWS_SDK_FOLDER/DLLs/Phone to
 cp $WINDOWS_SDK_FOLDER/SalesforceSDK/DLLs/Phone/x86/Salesforce.SDK.SmartSync.dll src/windows/WinMD/Phone/x86
```

Comment:
Do we still need these individual ```DLL``` copy steps, since we're now recursing through the ```SalesforceSDK``` folder?